flatpak/io.github.qtox.qTox.json

```diff
@@ -96,7 +96,6 @@
         {
           "type": "git",
           "url": "https://github.com/TokTok/qTox",
-          "commit": "927b87da1fa87fa3accff2a069fb9a3090424f7a",
```

Comment:
hmmm, not a fan

Reply:
Yeah, we need to find a way to do this in a stable way. The problem is while we're building the release, we don't know what that number is going to be. Probably this json should just contain the localised (directory) version of qtox, and we have the flathub version elsewhere, synced with this one but only this part different.

Reply:
Done.